packages/platforms/react-native/android/build.gradle

```
@@ -36,4 +36,5 @@ repositories {
 
 dependencies {
   implementation 'com.facebook.react:react-native'
+  compileOnly("com.bugsnag:bugsnag-android-performance:+")
```

given that the `generate-react-native-fixture` defined the published version number, we could rather specify that here as well (and avoid possibly picking up a "real" version by accident)?

```suggestion
  compileOnly("com.bugsnag:bugsnag-android-performance:9.9.9")
```